pyproject.toml

```
@@ -1,3 +1,154 @@
+[build-system]
+requires = ["setuptools >= 61.1.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "optuna"
+description = "A hyperparameter optimization framework"
+authors = [
+  {name = "Takuya Akiba"},
+  {email = "akiba@preferred.jp"}
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Mathematics",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+requires-python = ">=3.7"
+dependencies = [
+  "alembic>=1.5.0",
+  "cmaes>=0.9.0",
+  "colorlog",
+  "numpy",
+  "packaging>=20.0",
+  "scipy>=1.7.0",
+  "sqlalchemy>=1.3.0",
+  "tqdm",
+  "PyYAML",  # Only used in `optuna/cli.py`.
+]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+benchmark = [
+  "asv>=0.5.0",
+  "botorch",
+  "cma",
+  "scikit-optimize",
+  "virtualenv"
+]
+"checking" = [
```

We can remove double quotes :eyes:

```suggestion
checking = [
```